pyproject.toml

```diff
@@ -46,9 +46,9 @@ exasol-sagemaker-extension = ">=0.10.0,<1.0.0"
 # 
 # See ticket https://github.com/exasol/notebook-connector/issues/153
 
-exasol-integration-test-docker-environment = "==3.2.0"
-exasol-script-languages-container-tool = ">=0.19.0"
-requests = ">=2.31.0,<2.32.0"
+exasol-integration-test-docker-environment = ">=3.2.0"
+exasol-script-languages-container-tool = ">=1.0.0, < 2.0.0"
```

Comment:
add a comment, that we stay with 1.* because python-extension-common is still limited to it